src/gwt/src/org/rstudio/studio/client/workbench/views/buildtools/BuildPane.java

```diff
@@ -95,11 +95,14 @@ public BuildPane(Commands commands,
       ensureWidget();
    }
 
-   private void setInstallTooltip() 
+   private void updateInstallTooltips() 
```

Comment:
Should this also update the icons for the separate Clean and Install commands menu items? (not sure if that's happening based on your screenshots)

Reply:
Does changing the description on the command itself help? E.g.

https://github.com/rstudio/rstudio/blob/1870ca0dd3c7d72ee58281b6ee82c6f77da96a80/src/gwt/src/org/rstudio/core/client/command/AppCommand.java#L430-L433

Reply:
(If we can't get this figured out that's fine)